solr_schema.xml

```diff
@@ -26,171 +26,278 @@ schema. In this case the version should be set to the next CKAN version number.
 (x.y but not x.y.z since it needs to be a float) -->
 <schema name="ckan" version="2.7">
 
-<types>
```

Comment:
Please split the update of the schema and the reformating of the schema into 2 commits, so the changes can better be seen.